metalog.js

```diff
@@ -44,9 +44,21 @@ const logTypes = types => {
   return flags;
 };
 
-const normalizeStack = stack => stack.replace(/\s+at\s+/g, '\n\t');
+const normalizeStack = stack => {
+  const index = stack.indexOf(' at ');
+  return index === -1 ? stack : stack.slice(0, index).trimRight()
+    .concat('\n\t', stack.slice(index + 3).trimLeft());
+};
 
-const lineStack = stack => stack.replace(/[\n\r]\s*/g, '; ');
+const lineStack = stack => {
+  const indexes = [stack.indexOf('\n'), stack.indexOf('\r')];
```

Comment:
AFAIU, actual index of `\r` would be changed after slicing a stack.

Also, please rebase this on master and run both `npm install` and `npm run fmt`.